var/spack/repos/builtin/packages/lhapdf/package.py

```diff
@@ -32,3 +32,9 @@ class Lhapdf(AutotoolsPackage):
     def configure_args(self):
         args = ['FCFLAGS=-O3', 'CFLAGS=-O3', 'CXXFLAGS=-O3']
         return args
+
+    def patch(self):
```

Comment:
Hi @haralmha there appears to be a version 6.4.0 on the download website: do you know if they have patched that to work with Python >= 3.8? If so, IMO it would be preferable to resolve this by amending the `depends_on` constraints like


```
depends_on(python@:3.7, when=@:6.3)
depends_on(python@3.8:, when=@6.4:)
```

If not (or if you need 6.3.0 to work with Python >= 3.8 for your own purposes) then this patch looks good.